tools/helper/src/client/utils/getHeaders.ts

```diff
@@ -0,0 +1,132 @@
+export interface Header {
+  /**
+   * The level of the header
+   *
+   * `1` to `6` for `<h1>` to `<h6>`
+   */
+  level: number
+  /**
+   * The title of the header
+   */
+  title: string
+  /**
+   * The slug of the header
+   *
+   * Typically the `id` attr of the header anchor
+   */
+  slug: string
+  /**
+   * Link of the header
+   *
+   * Typically using `#${slug}` as the anchor hash
+   */
+  link: string
+  /**
+   * The children of the header
+   */
+  children: Header[]
+}
+
+export type HeaderLevels = false | number | [number, number] | 'deep'
+
+export type MenuItem = Omit<Header, 'slug' | 'children'> & {
+  element: HTMLHeadElement
+  children?: MenuItem[]
+}
+
+export interface GetHeadersOptions {
+  /**
+   * The selector of the headers
+   *
+   * @default '#vp-content :where(h1,h2,h3,h4,h5,h6)'
+   */
+  selector?: string
+  /**
+   * Ignore specific elements within the header.
+   *
+   * @default []
+   */
+  ignore?: string[]
+  /**
+   * The levels of the headers
+   *
+   * - `false`: No headers.
+   * - `number`: only headings of that level will be displayed.
+   * - `[number, number]: the first number is the minimum level and the second number is the maximum level.
```

Comment:
The high and low variables  Seems to be the reverse logic here.

I understand that inthe document tree h 2 seems to be higher level of h3,  But for the numbers, two is lower than three. And that is usually what we compare of because we are setting 213, not the actual teg names,

So I still suggest  Adjusting those two variable names to rename them as min or max